frontend/src/core/api/user.js

```diff
@@ -28,6 +28,35 @@ export default class UserAPI extends APIBase {
         return await this.fetch('get-users', 'GET', null, headers);
     }
 
+    /**
+     * Log UX action.
+     */
+    async logUxAction(
```

Comment:
I don't think this needs to be async, but it can just be a sync function returning a promise. The `return await` is the only await, and it seems unnecessary.